q2_metadata/_random.py

```diff
@@ -14,7 +14,10 @@
 def shuffle_groups(metadata: qiime2.CategoricalMetadataColumn,
                    n_columns: int = 3,
                    column_name_prefix: str = 'shuffled.grouping.',
-                   column_value_prefix: str = 'fake.group.') -> pd.DataFrame:
+                   column_value_prefix: str = 'fake.group.',
+                   sample_size: int = 1,
```

Comment:
My understanding of the goal for this one is that we'd encode the number of samples in each "fake group" in the name of the "fake group", so I think this parameter wouldn't be needed - only the new `encode_sample_size` parameter is needed. @hagenjp, @lizgehret - let me know if you have questions about that (and apologies if you already covered this in your meeting yesterday).

Reply:
@gregcaporaso @lizgehret I have removed the encode_sample_size parameter and the column_value_prefix parameter. Now the sample sizes are encoded and if 'sample_size' parameter is left blank the default is 1. For example, the column ids for a sample size of 3 would be 3.0, 3.1 , 3.2. Please let me know what you think about this option. Thanks!

Reply:
Hey @hagenjp! There might still be a little confusion here on the new functionality - we don't want to remove the `column_value_prefix` parameter, we just want to add in an optional flag (`encode_sample_size`) that will append the number of samples within each fake group to the end of the fake group name when enabled.

Check out @cherman2's comment [here](https://github.com/qiime2/q2-metadata/issues/57#issuecomment-1577367671) on the original Github issue - she provides a really great explanation on this, along with an example of what we expect the new behavior to look like. Hopefully this helps! Let me know if you have any other questions 🙂

Reply:
Okay @lizgehret, I think I see my mistake. My first attempt achieved the desired label but included a sample_size parameter. Am I correct that Greg is asking to remove?

Reply:
@hagenjp yes, we want to keep the `encode_sample_size` boolean parameter, but the numerical `sample_size` parameter is not needed. The sample size that gets appended to the end of the fake group name (when `encode_sample_size` is set to True) will be determined by the number of samples present within each group in the metadata. So that portion of the code change still needs to be implemented after updating the parameter that's being included in the method. Let me know if you'd like to set up another meeting to discuss this in greater detail!